packages/nomad/logs-collector.hcl

```diff
@@ -136,21 +136,22 @@ teamID = "{{ teamID }}"
 envID = "{{ envID }}"
 sandboxID = "{{ sandboxID }}"
 
-[sinks.grafana]
-type = "loki"
-inputs = [ "add_source_envd" ]
-endpoint = "${var.grafana_logs_endpoint}"
-encoding.codec = "json"
-auth.strategy = "basic"
-auth.user = "${var.grafana_logs_username}"
-auth.password = "${var.grafana_api_key}"
-
-[sinks.grafana.labels]
-source = "logs-collector"
-service = "{{ service }}"
-teamID = "{{ teamID }}"
-envID = "{{ envID }}"
-sandboxID = "{{ sandboxID }}"
+# [sinks.grafana]
```

Comment:
We might probably also cleanup the vars that were passed here from TF that might no longer be used before this goes out of draft.

Reply:
Good call, cleaned up